src/pages/messages/components/NviApprovals.tsx

```diff
@@ -79,21 +52,46 @@ const InstitutionApprovalStatusRow = ({ approvalStatus }: InstitutionApprovalSta
     gcTime: 1_800_000,
   });
 
-  const institutionAcronym = institutionQuery.data?.acronym ?? '';
+  return (
+    <TableRow>
+      <TableCell>
+        {institutionQuery.isPending ? (
+          <Skeleton />
+        ) : (
+          <Typography>{institutionQuery.data?.acronym ?? getLanguageString(institutionQuery.data?.labels)}</Typography>
+        )}
+      </TableCell>
+      <TableCell>
+        <InstitutionStatus status={approvalStatus.status} />
+      </TableCell>
+      <TableCell align="right">
+        <PublicationPointsTypography sx={{ whiteSpace: 'nowrap' }} points={approvalStatus.points} />
+      </TableCell>
+    </TableRow>
+  );
+};
+
+interface InstitutionStatusProps {
+  status: NviCandidateStatus;
+}
+
+const InstitutionStatus = ({ status }: InstitutionStatusProps) => {
+  const { t } = useTranslation();
+
+  const iconProps: SxProps = { mr: '0.2rem' };
+  const icon =
+    status === 'New' || status === 'Pending' ? (
+      <HourglassEmptyIcon fontSize="small" sx={iconProps} />
+    ) : status === 'Approved' ? (
+      <CheckIcon fontSize="small" sx={iconProps} />
+    ) : status === 'Rejected' ? (
+      <DoNotDisturbIcon fontSize="small" sx={iconProps} />
+    ) : null;
 
   return (
-    <>
-      {institutionQuery.isPending ? (
-        <Skeleton sx={{ width: '8rem' }} />
-      ) : (
-        <Typography>
-          {institutionAcronym ? institutionAcronym : getLanguageString(institutionQuery.data?.labels)}
-        </Typography>
-      )}
-      <Typography sx={{ whiteSpace: 'nowrap', justifySelf: 'center' }}>
-        {t(`tasks.nvi.status.${approvalStatus.status}`)}
-      </Typography>
-      <PublicationPointsTypography sx={{ whiteSpace: 'nowrap' }} points={approvalStatus.points} />
-    </>
+    <Box sx={{ display: 'flex' }}>
```

Comment:
Enda bedre enn å legge på margin til hvert ikon er å bruk gap på flexen (og ta vekk `iconProps`) 🎉 
```suggestion
    <Box sx={{ display: 'flex', gap: '0.2rem' }}>
```

Reply:
Lurt!